src/app/_components/Event/config.js

```diff
@@ -0,0 +1,15 @@
+export const monthsAhead = 5;
+export const monthNames = [
+  { index: 0, name: 'січень', en: 'january' },
+  { index: 1, name: 'лютий', en: 'february' },
+  { index: 2, name: 'березень', en: 'march' },
+  { index: 3, name: 'квітень', en: 'april' },
+  { index: 4, name: 'травень', en: 'may' },
+  { index: 5, name: 'червень', en: 'june' },
+  { index: 6, name: 'липень', en: 'july' },
+  { index: 7, name: 'серпень', en: 'august' },
+  { index: 8, name: 'вересень', en: 'september' },
+  { index: 9, name: 'жовтень', en: 'october' },
+  { index: 10, name: 'листопад', en: 'november' },
+  { index: 11, name: 'грудень', en: 'december' },
+];
```

Comment:
@Yivgha it's up to you, to create separate file, but if it used only in Event component, I would add it in the component file for the locality.

Reply:
changed